### PR TITLE
bowling: Add exercise

### PR DESCRIPTION
**Pull request by vmichalak**
My last contribution was on 2019.
Don't hesitate to make a big review :)

https://forum.exercism.org/t/bowling-add-exercise/9142

### CODE REVIEW

File: exercises/practice/bowling/src/test/kotlin/BowlingGameTest.kt [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        return game
+    }
+
+    @Test
+    fun `should be able to score a game with all zeros`() {
+        val game = playGame(intArrayOf(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
+        assertEquals(0, game.score())
+    }
+
+    @Test
```
Comment (ErikSchierboom) [inline exercises/practice/bowling/src/test/kotlin/BowlingGameTest.kt:20]: All but the first test should be ignored by default.

```suggestion
    @Test
    @Ignore
```

**Review by ErikSchierboom — COMMENTED** [review]
Thanks a lot! CI is failing due to there not being an entry for the exercise in the root `config.json`. See https://github.com/exercism/kotlin/actions/runs/7513491573/job/20455310442

File: exercises/practice/bowling/src/test/kotlin/BowlingGameTest.kt (hunk 1 above)
Comment (vmichalak) [reply exercises/practice/bowling/src/test/kotlin/BowlingGameTest.kt:21]: Corrected

File: exercises/practice/bowling/.meta/config.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "authors": [],
+  "contributors": [
+    "vmichalak"
+  ],
```
[the remark this answers is not in the document]
Comment (vmichalak) [reply exercises/practice/bowling/.meta/config.json]: Corrected

**Review by ErikSchierboom — APPROVED** [review]
Great work!

**Pull request merged**